### PR TITLE
Conformance result for v1.14/inspur-incloud-amd64

### PR DESCRIPTION
**Pull request by packyzbq**
This is a merge request for the conformance of product "Inspur Incloud K8s"

### CODE REVIEW

File: v1.14/inspur-incloud-amd64/PRODUCT.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+vendor: Inspur
+name: InCloud K8S
```
Comment (dankohn) [inline v1.14/inspur-incloud-amd64/PRODUCT.yaml]: Could you please rename to K8s? See https://github.com/cncf/foundation/blob/master/style-guide.md.

Comment (packyzbq) [reply v1.14/inspur-incloud-amd64/PRODUCT.yaml]: Hi, I've renamed the production name for amd64 and arm64.

**Comment by LambertShi001** [comment]
There was existing inspur arm architecture k8s PR:
https://github.com/cncf/k8s-conformance/pull/635

There is no need for repeated authentication.

**Comment by packyzbq** [comment]
> There was existing inspur arm architecture k8s PR:
> #635
> 
> There is no need for repeated authentication.

Hi, ICKS and the previous #635  are different products from different subsidiaries of Inspur. For our company, this authentication is still needed.

**Comment by taylorwaggoner** [comment]
@packyzbq thank you for submitting.  Please send in a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form) for your product(s).  Thank you!

**Comment by packyzbq** [comment]
> @packyzbq thank you for submitting. Please send in a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form) for your product(s). Thank you!

Hi, we have sent the participation form, please check.

**Comment by crystal-qw** [comment]
@taylorwaggoner Inspur has completed cncf certification, all services are based on container development, no need to do this certification。

**Comment by crystal-qw** [comment]
@taylorwaggoner As far as I know, inspur cloud is the sponsor of CNCF, and Inspur Open Platform is the only product of inspur container cloud.

**Comment by donaldliu** [comment]
@crystal-qw I am Donald Liu, a CNCF staff.  Can you let me know which company you represent?

**Comment by Seagullfy** [comment]
@taylorwaggoner Hi，Inspur has only one container cloud and has been certified. The results of the certification are as follows，https://github.com/cncf/k8s-conformance/pull/635&https://github.com/cncf/k8s-conformance/pull/636

**Comment by dankohn** [comment]
See https://github.com/cncf/k8s-conformance/pull/660#issuecomment-515718699

**Comment by donaldliu** [comment]
I have reached out the 2 companies of Inspur Group and they agree that we can all ignore comments from unrelated or unknown GitHub Ids ("crystal-qw" and "Seagullfy") and can continue processing the PR.  Please re-open this PR or suggest otherwise.

**Comment by dankohn** [comment]
OK, thanks for investigating.

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes
https://landscape.cncf.io/format=card-mode&selected=inspur-icks

**Pull request merged**